src/e2e-test/features/spanner/source/SpannerToSpanner_Additional.feature

```diff
@@ -0,0 +1,118 @@
+@Spanner_Source @SPANNER_TEST
+Feature: Spanner source - Verification of Additional Spanner to Spanner successful data transfer without using connections
+
+  @SPANNER_TEST @EXISTING_SPANNER_SINK
```

Comment:
@SourceRequired Tag is missing, Please identify if this test is also required or not.

Reply:
as per discussion with Ankit, @Spanner_Source will include this test . (?)